INSTALL.md

```diff
@@ -88,16 +88,20 @@ place for describing build procedures. Please do not remove this notice.
 QGIS, like a number of major projects (e.g., KDE 4.0),
 uses [CMake](https://www.cmake.org) for building from source.
 
-Following a summary of the required dependencies for building:
+We present the required and optional dependencies for building, using
+the upstream package names, rather than names within any particular
+packaging system.
 
-Required build tools:
+Required build tools (not needed at runtime):
 
+* C++ compiler supporting C++17
 * CMake >= 3.12.0
+* GNU make >= 4.3 (perhaps lower)
```

Comment:
is this actually required?

Reply:
I wrote in pkgsrc quite a while ago:
```
# Due to gmake spaces in filenames, we must use gmake.  An example is
# "resources/function_help/json/Date and Time".
# Further, gmake expects files that contain a $ such as "$area" to
# appear, unsurprisingly, as "$$area" on the RHS of dependency rules.
# NetBSD make, expects "$$$$area".  While cmake 3.17 produced
# "$$area", cmake 3.18 produces "$$$$area", perhaps only on NetBSD.
```

However, a fair question, and I am re-running the package build without gmake, which means BSD make will be used, and I'll comment again and/or change.

Reply:
It seems there are issues with BSD make still (build of 3.28.1 in pkgsrc context).  The generated makefiles have `$$area` (not sure why).  Also the quoting of files with spaces is treated differently.   Generally if an upstream uses GNU make features, I just mark the package to use it, rather than asking that it be avoided, as it doesn't seem like a good use of upstream development time to avoid a dependency that is not even a milli-QT.
```
[  0%] Built target version
make[2]: don't know how to make /tmp/work/geography/qgis/work/qgis-3.28.1/resources/function_help/json/rea (continuing)
make[2]: don't know how to make /tmp/work/geography/qgis/work/qgis-3.28.1/resources/function_help/json/urrentfeature (continuing)
make[2]: don't know how to make /tmp/work/geography/qgis/work/qgis-3.28.1/resources/function_help/json/ace_area (continuing)
make[2]: don't know how to make /tmp/work/geography/qgis/work/qgis-3.28.1/resources/function_help/json/ace_index (continuing)
make[2]: don't know how to make /tmp/work/geography/qgis/work/qgis-3.28.1/resources/function_help/json/eometry (continuing)
make[2]: don't know how to make /tmp/work/geography/qgis/work/qgis-3.28.1/resources/function_help/json/d (continuing)
make[2]: don't know how to make /tmp/work/geography/qgis/work/qgis-3.28.1/resources/function_help/json/ength (continuing)
make[2]: don't know how to make /tmp/work/geography/qgis/work/qgis-3.28.1/resources/function_help/json/ap (continuing)
make[2]: don't know how to make /tmp/work/geography/qgis/work/qgis-3.28.1/resources/function_help/json/erimeter (continuing)
make[2]: don't know how to make /tmp/work/geography/qgis/work/qgis-3.28.1/resources/function_help/json/ownum (continuing)
make[2]: don't know how to make /tmp/work/geography/qgis/work/qgis-3.28.1/resources/function_help/json/cale (continuing)
make[2]: don't know how to make /tmp/work/geography/qgis/work/qgis-3.28.1/resources/function_help/json/ertex_as_point (continuing)
make[2]: don't know how to make /tmp/work/geography/qgis/work/qgis-3.28.1/resources/function_help/json/ertex_index (continuing)
make[2]: don't know how to make /tmp/work/geography/qgis/work/qgis-3.28.1/resources/function_help/json/ertex_x (continuing)
make[2]: don't know how to make /tmp/work/geography/qgis/work/qgis-3.28.1/resources/function_help/json/ertex_y (continuing)
make[2]: don't know how to make /tmp/work/geography/qgis/work/qgis-3.28.1/resources/function_help/json/ertex_z (continuing)
make[2]: don't know how to make /tmp/work/geography/qgis/work/qgis-3.28.1/resources/function_help/json/_at (continuing)
make[2]: don't know how to make /tmp/work/geography/qgis/work/qgis-3.28.1/resources/function_help/json/Date\ (continuing)
make[2]: don't know how to make and\ (continuing)
make[2]: don't know how to make Time (continuing)

```

Reply:
It's just that "make" **isn't** a universal requirement for building QGIS. So if it's needed for BSD then it should be listed only under BSD build instructions, not in this section.

Reply:
Ninja works very well

Reply:
I get it now about cmake generator options; will rewrite to make sense in that context and force-push.  Thanks for the comments.